test/unit/accord/utils/Gens.java

```diff
@@ -372,7 +372,12 @@ public ListDSL(Gen<T> fn) {
         @Override
         public ListDSL<T> unique()
         {
-            return new ListDSL<>(new GenReset<>(fn));
+            return new ListDSL<>(new GenReset<>(fn, false));
+        }
+
+        public ListDSL<T> uniqueBestEffort()
```

Comment:
not for this patch: going to have to do the same in accord... before we merge accord to trunk we are going to have to account for the drift between the 2 repos....